installation-bundle/src/DependencyInjection/ContaoInstallationExtension.php

```diff
@@ -46,9 +46,17 @@ public function load(array $configs, ContainerBuilder $container): void
      */
     public function prepend(ContainerBuilder $container): void
     {
-        $configDir = $container->getParameter('kernel.project_dir').'/app/config';
+        $projectDir = $container->getParameter('kernel.project_dir');
 
-        if (file_exists($configDir.'/parameters.yml') || !file_exists($configDir.'/parameters.yml.dist')) {
+        if (file_exists($projectDir.'/config/parameters.yml') || file_exists($projectDir.'/app/config/parameters.yml')) {
+            return;
+        }
+
+        if (file_exists($projectDir.'/config/parameters.yml.dist')) {
```

Comment:
Again, this file can never exist.

Reply:
see eba9e707bb95c2dc6fb00c9ec7c18fab2b91935f